client/src/pages/Settings/Settings.tsx

```diff
@@ -28,7 +29,7 @@ const settingsMenu = [
   {
     name: 'Payment methods',
     to: '/profile/settings/payment-methods',
-    component: <SettingHeader header="Payment Methods" />,
+    component: <StripeConnect />,
```

Comment:
This isn't for payment methods, let's just make a new settings page called 'Billing' or 'Stripe'